lib/utils.py

```diff
@@ -108,6 +107,26 @@ def set_extras_n_fix_units(
 
     args["resource_provides_quoted"] = [fixquote(x) for x in args["resource_provides"]]
 
+    # if the user defined the usage_model on the command line,
+    # we need to use their definition of usage_model, not ours
+    # also, we define the site as 'Fermigrid' if they have not requested OFFSITE
+    add_site = ""
+    for r in args["resource_provides_quoted"]:
+        if "usage_model" in r:
```

Comment:
This line actually uncovered a mistake in our unit tests.  I'll open a PR for that, but it shouldn't conflict with your code.

Reply:
fixed

Reply:
ok